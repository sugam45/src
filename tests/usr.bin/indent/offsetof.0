/*	$NetBSD: offsetof.0,v 1.1 2019/04/04 15:27:35 kamil Exp $	*/
/* $FreeBSD: head/usr.bin/indent/tests/offsetof.0 313544 2017-02-10 09:31:39Z pstef $ */
/* See r303718 */
void t(void) {
	int n = malloc(offsetof(struct s, f) + 1);
}
