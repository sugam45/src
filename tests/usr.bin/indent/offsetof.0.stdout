/*	$NetBSD: offsetof.0.stdout,v 1.1 2019/04/04 15:27:35 kamil Exp $	*/
/* $FreeBSD: head/usr.bin/indent/tests/offsetof.0.stdout 321381 2017-07-23 14:04:45Z pstef $ */
/* See r303718 */
void
t(void)
{
	int		n = malloc(offsetof(struct s, f) + 1);
}
